packages/rest/src/rest-server.ts

```diff
@@ -132,14 +126,8 @@ export class RestServer extends Context implements Server {
     };
 
     this.bind(RestBindings.HANDLER).toDynamicValue(() => this.httpHandler);
-
-    this.bind(SequenceActions.FIND_ROUTE).toProvider(FindRouteProvider);
     this.bind(SequenceActions.PARSE_PARAMS).to(parseOperationArgs);
-    this.bind(SequenceActions.INVOKE_METHOD).toProvider(InvokeMethodProvider);
     this.bind(SequenceActions.SEND).to(writeResultToResponse);
```

Comment:
Please bind `SEND` at application level too (I think you will need to write a dummy provider for that).